client/src/pages/scheduling/Confirm.js

```diff
@@ -0,0 +1,149 @@
+  },
+}));
+
+const Confirm = (props) => {
+  const theme = useTheme();
+  const matches = useMediaQuery(theme.breakpoints.down("sm"));
+  const classes = useStyles();
+
+  const eventTypeID = props.match.params.event_type;
+  const [eventType, setEventType] = useState(null);
+  const [name, setName] = useState("");
+  const [email, setEmail] = useState("");
+
+  const timeslot = props.match.params.timeslot;
+
+  function initialFetch(eventTypeID) {
+    axios.get(`http://localhost:3001/api/meeting_types/single/${eventTypeID}`)
+      .then(response => setEventType(response.data));
+  }
+
+  useEffect(() => {
+    initialFetch(eventTypeID);
+  }, []);
+
+  const handleNameChange = (event) => setName(event.target.value);
+  const handleEmailChange = (event) => setEmail(event.target.value);
+
+  const handleSubmit = (event) => event.preventDefault();
+
+
+  return (
+    <Container>
+      <Grid container justify="flex-end">
+        <Grid container justify="flex-end" item xs={4}>
+          <Button 
+            className={classes.nav} variant="outlined" 
+            href="http://localhost:3000/event_types/user/me">Home</Button>
+        </Grid>
+      </Grid>
+      {eventType && 
+      <Paper className={classes.root}>
+        <Box p={4}>
+          <Grid container spacing={2}>
+            <Grid item md={3} xs={12}>
+              <Typography variant="subtitle1" align={matches ? "center" : "inherit"} gutterBottom>{eventType.user.firstName} {eventType.user.lastName}</Typography>
+              <Typography variant="h5" align={matches ? "center" : "inherit"} gutterBottom>{eventType.name}</Typography>
+              <Grid container spacing={1} alignItems="center" justify={matches ? "center" : "flex-start"}>
+                <Grid item>
+                  <Timer />
+                </Grid>
+                <Grid item>
+                  <Typography variant="subtitle2">{eventType.duration} min</Typography>
+                </Grid>
+              </Grid>
+              <Grid container spacing={1} alignItems="center" justify={matches ? "center" : "flex-start"}>
+                <Grid item>
+                  <EventAvailable />
+                </Grid>
+                <Grid item>
+                  <Typography variant="subtitle2">{(new Date(timeslot)).toLocaleString()}</Typography>
+                </Grid>
+              </Grid>
+              <Grid container spacing={1} alignItems="center" justify={matches ? "center" : "flex-start"}>
+                <Grid item>
+                  <Public />
+                </Grid>
+                <Grid item>
+                  <Typography variant="subtitle2">{Intl.DateTimeFormat().resolvedOptions().timeZone}</Typography>
+                </Grid>
+              </Grid>
+            </Grid>
+            <Divider orientation={matches ? "horizontal" : "vertical"} flexItem={matches ? false : true} fullWidth />
+            <Grid container item md={9} xs={12} spacing={1} >
+              <Grid item md={12} xs={12}>
```

Comment:
```suggestion
              <Grid item xs={12}>
```

since these are the same, you just need to explicitly determine the smallest scale (`xs`)